engine/engine-core/src/main/scala/org/apache/spark/frame/FrameRdd.scala

```diff
@@ -496,6 +496,26 @@ object FrameRdd {
   }
 
   /**
+   * Converts a spark dataType (as string)to our schema Datatype
```

Comment:
Let's not put Datatype conversion methods here. Use parseMany routine from DataTypes.scala. Same applies for LoadHBaseImpl.scala. We should fix both.